src/ActionBar/ActionBar.js

```diff
@@ -27,6 +27,11 @@ ActionBar.propTypes = {
     width: PropTypes.string
 };
 
+ActionBar.defaultProps = {
+    mobile: false,
```

Comment:
Looks like there are just a couple default props set to false remaining:

* `ActionBar`: `mobile`
* `TimeItem`: `disabled`, `format12hours `, `spinners `